R/plumber.R

```diff
@@ -443,7 +443,15 @@ Plumber <- R6Class(
               TRUE
             } else {
               # there are other endpoints, so get only nodes with name ""
-              names(node) == ""
+              # which path does not end with / and path is not root
+              node_path <- function(node) {
+                if (!is.null(node$path)) {
+                  return(node$path)
+                } else {
+                  return(character(1))
+                }
```

Comment:
```suggestion
                node$path %||% ""
```

Reply:
what is this black magic operator, I want in.